frodo.js

```diff
@@ -1,20 +1,20 @@
-/**
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?